### ISSUE TITLE
Statistics Plugin doesn't show up in the statistics window

### ISSUE THREAD

**Issue by thorstenhenke** [gephi/gephi-plugins#213]
Dear gephi developers, 

I am currently trying to develop a plugin that calculates a node level statistic. I carefully read the gephi-plugins README.md and also had a close look at the statistics' related examples in the gephi-plugin-bootcamp. 

To get familiar with maven, netbeans and the building process I created a mockup version of my plugin and tried to run in. Although gephi recognizes the plugin and lists it as 'installed' the plugin's method does not show up in the statistics window. 

Since I followed the instructions on how to build a package very caerfully and by now my plugin is just a mockup version, I am out of ideas how to solve the problem by myself. 

_Update_
When I run the plugin from within netbeans I get the following warining messsage. 
`java.lang.ClassNotFoundException: Will not load classes from default package (IOIndexUI)`

Another interesting obersevation is, that I can import  my plugin in my regular Gephi installation and also see it listed in the statistics window. I presume based on the warning message and the my experience with the regular installation that there is s

You can find my current plugin implementation here:
https://github.com/thorstenhenke/gephi-plugins

I am clearly missing out a minor but important detail...

**Comment by AnshulV98** [comment]
I am not sure about it, as I cannot see your code, But is it possible that you may have not created a StatisticsUI class for your module? I believe that might be required for the plugin to show up in the Statistics Tab. https://github.com/gephi/gephi/wiki/Statistics maybe this might help?

**Comment by AnshulV98** [comment]
@thorstenhenke I tried this myself and faced a similar issue, but by fiddling around I realized that if one puts all the java files in a package then it simply shows up!!! your 
```
java.lang.ClassNotFoundException: Will not load classes from default package (IOIndexUI)

```
was a big help for this. Also, don't forget to have appropriate Manifest.mf files and Bundle.properties files for your plugin.

Have a look at 
https://github.com/carlschroedl/gephi-plugins/tree/minimum-spanning-tree-plugin/modules/MinimumSpanningTree/src/main f 

for reference, this helped me make it work as well. Hope this helps
